gdx/src/com/badlogic/gdx/graphics/g2d/BitmapFont.java

```diff
@@ -515,7 +362,7 @@ public int computeVisibleGlyphs (CharSequence str, int start, int end, float ava
 
 		for (; index < end; index++) {
 			char ch = str.charAt(index);
-			if (ch == '[' && markupEnabled) {
+			if (ch == '[' && data.markupEnabled) {
 				index++;
```

Comment:
This is the issue mentioned by @MobiDevelop in https://github.com/libgdx/libgdx/issues/2886#issuecomment-83352300

Looks like this code has not changed drastically. So, if this PR is not merged soon I can fix and test the issue using the snapshot, then I can paste the fix here. Objections?